lib/rubocop/cop/rspec/spec_file_path_format.rb

```diff
@@ -32,6 +32,11 @@ module RSpec
       #   # good
       #   whatever_spec.rb         # describe MyClass, type: :routing do; end
       #
+      # @example `IgnoreMetadata: {type=>[routing,models]}` (default)
+      #   # good
+      #   whatever_spec.rb         # describe MyClass, type: :routing do; end
+      #   whatever_spec.rb         # describe MyClass, type: :models do; end
```

Comment:
What if the metadata's value IS an array?
```ruby
RSpec.describe Airplane, prepare: [:fuel] do
```
How do we define IgnoreMetadata in YAML?